### PR TITLE
fix(coinjoin): skip round if selected utxo effective value < allowedOutputAmounts.min

### PR DESCRIPTION
**Pull request by szymonlesisz**
## Description
temporary fix for [middleware coin selection algorithm ](https://github.com/zkSNACKs/WalletWasabi/issues/10759) which results with `NotEnoughFunds` error [reported in sentry](https://satoshilabs.slack.com/archives/C04SRD1G2BD/p1685564773566439)

this commit could/should be reverted when it's fixed

### CODE REVIEW

File: packages/coinjoin/tests/client/selectRound.test.ts [hunk 1]
```diff
@@ -463,6 +503,7 @@ describe('selectRound', () => {
         expect(spy).toBeCalledTimes(9);
 
         ['AA', 'AB', 'AC', 'CA'].forEach((outpoint, index) => {
+            console.warn('RESULT', result!.inputs[index], index);
```
Comment (matejkriz) [inline packages/coinjoin/tests/client/selectRound.test.ts]: leftover?

Comment (szymonlesisz) [reply packages/coinjoin/tests/client/selectRound.test.ts]: ahh, right :+1:

**Comment by nopara73** [comment]
Heyhey, for managing expectations, I just wanted to note that this is the next thing on my plate, so on the Wasabi side the fix is a few days away.

**Comment by szymonlesisz** [comment]
@nopara73 thanks for the info :+1: 

We have upcoming suite release (there is a freeze next tuesday, then it will be tested by QA) and i dont know if you manage to fix it before that date.

Since it happens quite often on production (like once per day, maybe more) we decided to add this as a hotfix and sure we will remove it after your fix

**Review by matejkriz — APPROVED** [review]
👍 please resolve the conflict in test and merge

**Pull request merged**